.travis.yml

```diff
@@ -1,7 +1,12 @@
-language: python
+# We set the language to c because python isn't supported on the MacOS X nodes
+# on Travis. However, the language ends up being irrelevant anyway, since we
+# install Python ourselves using conda.
+language: c
```

Comment:
I was wondering about that, because it seemed like Travis was trying to install a system python, and I was pretty sure we were using miniconda's python.  I think this makes more sense to put `c` here (or maybe `bash`)?

Reply:
I would stick with C since it works for Astropy. :)